packages/babel-helpers/src/helpers/defineEnumerableProperties.ts

```diff
@@ -14,13 +18,13 @@ export default function _defineEnumerableProperties(obj, descs) {
   // Symbols are available, fetch all of the descs object's own
   // symbol properties and define them on our target object too.
   if (Object.getOwnPropertySymbols) {
-    var objectSymbols = Object.getOwnPropertySymbols(descs);
-    for (var i = 0; i < objectSymbols.length; i++) {
-      var sym = objectSymbols[i];
+    var objectSymbols: Array<Symbol> = Object.getOwnPropertySymbols(descs);
+    for (const sym of objectSymbols) {
```

Comment:
The babel 7 helpers should be ES5-compliant, so we should stick to the good old for loop.